react/features/screenshot-capture/ScreenshotCaptureSummary.tsx

```diff
@@ -30,7 +30,11 @@ declare let ImageCapture: any;
 export default class ScreenshotCaptureSummary {
     _state: IReduxState;
     _initializedRegion: boolean;
+
+    // @ts-ignore
```

Comment:
Remove.

Reply:
Available for all the ts-ignores I added in this PR, seems that some of the changes here have affected ts and new errors have risen.

Reply:
Please drop them. I want to see the CI error. I think it's a local error you have. You haven't touched anything related to screenshot-capture even remotely, in this PR.